specification/schemas/Organization.json

```diff
@@ -23,6 +23,37 @@
         "iban": {
           "$ref": "#/components/schemas/Iban"
         },
+        "language": {
+          "type": "enum",
+          "enum": [
+            "english"
+          ],
+          "example": "english"
+        },
+        "payment_method": {
+          "type": "enum",
+          "enum": [
+            "sepa_direct_debit",
+            "on_credit",
+            "prepayment"
+          ],
+          "example": "prepayment"
+        },
+        "vat_number": {
+          "type": "string",
+          "example": "123456789MVA"
+        },
+        "payment_term": {
+          "type": "integer",
+          "example": "1"
```

Comment:
It would be nice to have a description here. Does this `1` mean a day or a month?